LEGO1/mxdirectdraw.cpp

```diff
@@ -1,22 +1,1223 @@
+
 #include "mxdirectdraw.h"
+#include "decomp.h"
 
-// OFFSET: LEGO1 0x1009e7f0
-int MxDirectDraw::FlipToGDISurface()
+
+#ifndef DDSCAPS_3DDEVICE
+#define DDSCAPS_3DDEVICE 0x00002000l
```

Comment:
At what times is this not defined? Shouldn't this always be in `ddraw.h`? If not, a comment should probably be added explaining it